### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by constantweb**
[DEMO LINK](https://constantweb.github.io/layout_calendar/)

### CODE REVIEW

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
add files to your pull request

**Comment by constantweb** [comment]
Які саме файли?

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__day calendar__day--start-day_fri"></div>
```
Comment (marina-tilniak) [inline src/index.html:11]: move this modifier to calendar

**Review by marina-tilniak — CHANGES_REQUESTED** [review]
you don't have  files with styles in GitHub
image

**Pull request closed without merging**